darts/timeseries.py

```diff
@@ -20,6 +20,7 @@
 class TimeSeries:
     def __init__(self,
                  df: pd.DataFrame,
+                 columns: Optional[Union[List[str], str]] = None,
```

Comment:
Super cool feature :) Could you put `columns` as last keyword argument? That way when this will be merged it won't break code for peoples that are using `freq` as 3rd argument or `fill_missing_dates` as 4th argument

Reply:
I already fixed it in many places in the library but true I could also make it as 4th argument for final users